source/agora/consensus/protocol/Nominator.d

```diff
@@ -14,6 +14,7 @@
 module agora.consensus.protocol.Nominator;
 
 import agora.common.crypto.Key;
+import agora.common.crypto.Schnorr;
```

Comment:
I think you got the `import` inverted, this commit requires `ECC`, the next one `Schnorr`.

Reply:
No it is the right way round. This one needs Schnorr Pair (which is actually a keyPair also)